build.gradle

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id 'groovy'
+    id 'maven-publish'
+}
+
+group = 'io.jenkins.infra'

```

Comment:
I guess it makes sense to use a group name with our values like 'com.wooga' or 'net.wooga'.

Reply:
Yeah, this was a copy leftover. Changed it.